tests/test_waypoints_to_commands.py

```diff
@@ -6,6 +6,17 @@
 
 from modules import waypoints_to_commands
 
+def test_waypoints_to_commands_empty_input():
+    """
+    Tests functionality correctness of waypoints_to_commands on empty input
+    """
+    waypoints = []
+    altitude = 100
+    
+    result, commands_actual = waypoints_to_commands.waypoints_to_commands(waypoints, altitude)
+    
+    assert not result
+    assert commands_actual is None
 
```

Comment:
2 empty lines between top level functions.

Reply:
Add a 2nd empty line between end of top level function and beginning of top level function.